Detectors/Base/include/DetectorsBase/MatLayerCylSet.h

```diff
@@ -19,9 +19,12 @@
 #include "DetectorsBase/MatLayerCyl.h"
 #include "DetectorsBase/Ray.h"
 #include "FlatObject.h"
+#include <vector>
```

Comment:
We should not include standard C / C++ headers for GPU code.
This should go below into `#ifndef GPUCA_ALIGPUCODE`

It can be used for initialization of the classes, but obviously not for accessing the MatLUT. All data that is needed for accessing the MatLUT must go into the FlatObject, which is transferred to GPU.